_includes/code/quickstart/endtoend.py

```diff
@@ -112,13 +118,18 @@
         filters=wvc.query.Filter.by_property("category").equal("ANIMALS")
     )
 
-    print(response.objects[0].properties)  # Inspect the first object
+    # Extract the properties of the first two objects
```

Comment:
As above, let's leave this as is for more stream lined code

Reply:
Ok